### PR TITLE
Add editor config file

### PR DESCRIPTION
**Pull request by oleh-momot**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+insert_final_newline = true
+
+# Indentation override for all JS
+[*.js]
+charset = utf-8
+indent_style = space
+indent_size = 4
+
+# Matches the exact files either package.json
+[*.json]
```
Comment (StyleT) [inline .editorconfig]: Add json5

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+
+# Indentation override for all JS
+[*.js]
```
Comment (StyleT) [inline .editorconfig]: Add ts/tsx

File: .editorconfig (hunk 1 above)
Comment (oleh-momot) [reply .editorconfig]: Added

File: .editorconfig (hunk 2 above)
Comment (oleh-momot) [reply .editorconfig]: Added

**Comment by StyleT** [comment]
Well done 😉

**Pull request merged**